nomad/leader.go

```diff
@@ -532,8 +532,10 @@ func (s *Server) reapFailedEvaluations(stopCh chan struct{}) {
 			// due to the fairly large backoff.
 			followupEvalWait := s.config.EvalFailedFollowupBaselineDelay +
 				time.Duration(rand.Int63n(int64(s.config.EvalFailedFollowupDelayRange)))
+
 			followupEval := eval.CreateFailedFollowUpEval(followupEvalWait)
 			updateEval.NextEval = followupEval.ID
+			updateEval.ModifyTime = time.Now().UTC().UnixNano()
```

Comment:
Also, `restoreEvals` in this file may need to set times if they're 0 because they were generated by an old version, if that's what we should do

Reply:
We shouldn't change createtime for older evals, that seems wrong and confusing. Better to leave them unset I think?